dev-client/App.tsx

```diff
@@ -5,34 +5,36 @@
  * @format
  */
 
-import React from 'react';
-import { StyleSheet, View } from 'react-native';
-
+import React, {useEffect} from 'react';
+import {NavigationContainer} from '@react-navigation/native';
 import Mapbox from '@rnmapbox/maps';
-import { Button, NativeBaseProvider, Text } from 'native-base';
-import { theme } from './theme';
-import LoginView from './components/Login';
-
-
+import {NativeBaseProvider} from 'native-base';
+import {theme} from './theme';
+import {LoginProvider} from './context/LoginContext';
+import ScreenDisplay from './screens/ScreenDisplay';
+import './translations';
+import {checkAndroidPermissions} from './native';
+import {PermissionsAndroid} from 'react-native';
 
 Mapbox.setAccessToken(
   'pk.eyJ1Ijoic2hyb3V4bSIsImEiOiJjbGY4bW8wbGEwbDJnM3FsN3I1ZzBqd2kzIn0.2Alc4o911ooGEtnObLpOUQ',
 );
 
-
 function App(): JSX.Element {
+  useEffect(() =>
+    checkAndroidPermissions(
+      PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION,
+    ),
+  );
   return (
     <NativeBaseProvider theme={theme}>
-      <View style={[styles.container, {
-        justifyContent: "center"
-      }]}>
-        { /* <Mapbox.MapView style={styles.map} /> */}
-        <LoginView />
-      </View>
+      <NavigationContainer>
+        <LoginProvider>
```

Comment:
LoginProvider is a context I introduced when I thought we would be doing SSO right away.